2.0/TransmissionTest/glTF/TransmissionTest.gltf

```diff
@@ -0,0 +1,1471 @@
+                0.0,
+                0.0,
+                0.7071068286895752
+            ],
+            "scale": [
+                0.9999997019767761,
+                0.9999998211860657,
+                0.3821372091770172
+            ],
+            "mesh": 18,
+            "name": "TransmissionTextureLabel"
+        },
+        {
+            "translation": [
+                0.4193941354751587,
+                0.8123779892921448,
+                2.6822088994293837e-9
+            ],
+            "rotation": [
+                0.7070714235305786,
+                -0.007068855222314596,
+                0.0070688542909920219,
+                0.7070714831352234
+            ],
+            "scale": [
+                1.002103567123413,
+                1.0000001192092896,
+                0.5380522012710571
+            ],
+            "mesh": 19,
+            "name": "TransmissionTexture&OpacityLabel"
+        }
+    ],
+    "samplers": [
+        {},
+        {},
+        {},
+        {},
+        {},
+        {},
+        {}
+    ],
+    "scenes": [
+        {
+            "nodes": [
+                0,
+                1,
+                2,
+                3,
+                4,
+                5,
+                6,
+                7,
+                8,
+                9,
+                10,
+                11,
+                12,
+                13,
+                14,
+                15,
+                16,
+                17,
+                18,
+                19
+            ],
+            "name": "scene"
+        }
+    ],
+    "textures": [
+        {
+            "name": "texture2492",
+            "sampler": 0,
+            "source": 0
+        },
+        {
+            "name": "texture14184",
+            "sampler": 1,
+            "source": 1
+        },
+        {
+            "name": "texture11360",
+            "sampler": 2,
+            "source": 2
+        },
+        {
+            "name": "texture4086",
+            "sampler": 3,
+            "source": 3
+        },
+        {
+            "name": "texture5402",
+            "sampler": 4,
+            "source": 4
+        },
+        {
+            "name": "texture10557",
+            "sampler": 5,
+            "source": 5
+        },
+        {
+            "name": "texture37253",
+            "sampler": 6,
+            "source": 6
+        }
+    ],
+    "scene": 0,
+    "extensions": {
+        "KHR_xmp": {
+            "@context": {
+                "dc": "http://purl.org/dc/elements/1.1/"
+            },
+            "packets": [
+                {
+                    "dc:date": "2019-10-25T15:25:01.695-07:00",
+                    "dc:format": "model/gltf+json",
+                    "dc:title": "ASM_transparency_labels.dn",
+                    "xmp:CreatorTool": "Adobe Dimension"
+                }
+            ]
+        }
+    },
+    "extensionsUsed": [
+        "KHR_xmp",
+        "KHR_texture_transform",
```

Comment:
Echoing Alex's concern from https://github.com/KhronosGroup/glTF/pull/1698#discussion_r442783701, I don't think the official sample model for transmission should depend on `KHR_texture_transform`.  Can it be removed from this model?  XMP is fine to keep, as it's optional, but texture transform is required here for the correct labels.

In other models that require it, Adobe Dimension should probably list `KHR_texture_transform` under `extensionsRequired` given that the texturing looks broken in viewers that don't support it.  Viewers shouldn't attempt to load the model at all if they can't process the texture transform extension.

Reply:
I'll work on this. Might be a couple of days.

Reply:
Okay, done.